Framework/src/PostProcessingConfig.cxx

```diff
@@ -50,6 +50,11 @@ PostProcessingConfig::PostProcessingConfig(const std::string& id, const boost::p
     stopTriggers.push_back(stopTrigger.second.get_value<std::string>());
   }
   auto ppTree = config.get_child("qc.postprocessing." + id);
+  if (ppTree.count("taskParameters") > 0) {
+    for (const auto& [key, value] : ppTree.get_child("taskParameters")) {
+      customParameters.set(key, value.get_value<std::string>());
+    }
+  }
```

Comment:
It's a different behaviour than for QC Tasks, where it's:
```
  if (taskTree.count("extendedTaskParameters") > 0) {
    ts.customParameters.populateCustomParameters(taskTree.get_child("extendedTaskParameters"));
  } else if (taskTree.count("taskParameters") > 0) {
    for (const auto& [key, value] : taskTree.get_child("taskParameters")) {
      ts.customParameters.set(key, value.get_value<std::string>());
    }
  }
```
In particular, the code for PP will merge `taskParameters` with `extendedTaskParameters`, while the code for QC tasks will take one or the other. Could you unify those for the sake of consistency?

Reply:
Good catch, this is not what I intended. Fixed.